vignettes/faq.Rmd

```diff
@@ -181,7 +181,7 @@ quantile(pred$sd, c(0.01, 0.1, 0.25, 0.5, 0.75, 0.9, 0.99))
 
 We recommend use of the [`priorsense`](https://github.com/n-kall/priorsense) package [@kallioinen2024detecting] to check how sensitive the posterior distribution is to perturbations of the prior distribution and likelihood using power-scaling analysis:
 
-```{r}
+```{r, eval = FALSE}
```

Comment:
Is this intentional? It won't produce the figure then?

Reply:
yes - see the comment in the PR issue it errors with the new likelihood